### ISSUE TITLE
[v1.7.0-rc2] glooctl check outputs an error message when checking secrets

### ISSUE THREAD

**Issue by bcollard** [solo-io/gloo#4519]
**Describe the bug**
From a fresh install, I've deployed httpbin and one VS.

```bash
[bco-testing-1.7-rc:default]:~/p/s/s/c/b/20210330-testing-1.7-rc (main✗) $ glooctl check                         [10:05:51]
Checking deployments... OK
Checking pods... OK
Checking upstreams... OK
Checking upstream groups... OK
Checking auth configs... OK
Checking rate limit configs... OK
Checking secrets... I0331 10:06:03.743826   53302 request.go:645] Throttling request took 1.19968024s, request: GET:https://192.168.1.99:1000/api/v1/namespaces/kube-node-lease/pods?limit=500&resourceVersion=0
OK
Checking virtual services... OK
Checking gateways... OK
Checking proxies... OK
Checking rate limit server... OK
No problems detected.
I0331 10:06:13.778375   53302 request.go:645] Throttling request took 1.245734249s, request: GET:https://192.168.1.99:1000/apis/flowcontrol.apiserver.k8s.io/v1beta1?timeout=32s
Skipping Gloo Instance check -- Gloo Federation not detected
```

`glooctl check`
```json
Client: {"version":"1.7.0-rc2"}
Server: {"type":"Gateway","enterprise":true,"kubernetes":{"containers":[{"Tag":"1.7.0-rc1","Name":"discovery","Registry":"quay.io/solo-io"},{"Tag":"1.7.0-rc2","Name":"extauth-ee","Registry":"quay.io/solo-io"},{"Tag":"1.7.0-rc1","Name":"gateway","Registry":"quay.io/solo-io"},{"Tag":"1.7.0-rc2","Name":"gloo-ee-envoy-wrapper","Registry":"quay.io/solo-io"},{"Tag":"1.7.0-rc2","Name":"gloo-ee","Registry":"quay.io/solo-io"},{"Tag":"1.7.0-rc2","Name":"observability-ee","Registry":"quay.io/solo-io"},{"Tag":"1.7.0-rc2","Name":"rate-limit-ee","Registry":"quay.io/solo-io"},{"Tag":"5","Name":"redis","Registry":"docker.io"}],"namespace":"gloo-system"}}
```

**Expected behavior**
No error

**Additional context**
Add any other context about the problem here, e.g.
- Gloo Edge version: v1.7.0-rc2
- Kubernetes version: kind 1.20.2

**Comment by kevin-shelaga** [comment]
Just want to add that the throttling happens against different resource types each run. Looks similar to when you pound istiod

`.../apis/cert-manager.io/v1?timeout=32s`
`.../apis/autoscaling/v2beta1?timeout=32s`
`.../apis/authorization.k8s.io/v1beta1?timeout=32s`
`.../api/v1/namespaces/istio-system/configmaps?limit=500&resourceVersion=0`

**Comment by jahvon** [comment]
The fix will be in Gloo v1.7.2
